### DISCUSSION TITLE
No hallucinations. Whisper + Flan-t5

### DISCUSSION THREAD

**Discussion by i-am-neo** [openai/whisper#534] [Show and tell]
![image](https://neo.vidd.ai/p/gifs/askneo_demo_whisper.gif)

Powered by Whisper and Flan-T5, Neo is an AI who creates a knowledge base from meetings.
Speaker identification is built in.

Handy for distributed teams...
More about [Neo here](https://neo.vidd.ai?from=gh_whisper).

**Comment by rajivpoddar** [comment]
Looks cool! How are you doing speaker diarization?
